src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -769,4 +769,11 @@ public static String getHorseArmorTexture(EntityHorse horse, ItemStack armorStac
         if(texture == null) texture = horse.getHorseArmorType().getTextureName();
         return texture;
     }
+
+    public static float getNightVisionBrightness(EntityRenderer renderer, Entity entity, IBlockState state, float partial, float brightness)
+    {
+        EntityViewRenderEvent.NightVisionBrightness event = new EntityViewRenderEvent.NightVisionBrightness(renderer, entity, state, partial, brightness);
+        if (MinecraftForge.EVENT_BUS.post(event)) return event.getBrightness();
```

Comment:
... whats the point of requiring it to be canceled instead of just changing the value?

Reply:
So that another mod doesn't get the event. However, looking at it again I think it could be removed.